tardis/io/tests/test_model_reader.py

```diff
@@ -5,31 +5,35 @@
 import pytest
 
 import tardis
-from tardis.io.model_reader import (read_artis_density, 
-read_simple_ascii_abundances)
+from tardis.io.model_reader import read_artis_density, \
```

Comment:
I like the old way better. In my opinion `\` should not be used for linebreaks in python.